### ISSUE TITLE
@nestjs/platform-fastify after update don't support application/x-www-form-urlencoded

### ISSUE THREAD

**Issue by txantslusam** [nestjs/nest#3936]
## Describe the regression
After upgrade of `@nestjs/platform-fastify` to version `6.11.1`, for all webhook requests from Paddle payments that are using `application/x-www-form-urlencoded`, nest is returning error `415`.

Response from version `6.11.1`:
```json
{
    "statusCode": 415,
    "code": "FST_ERR_CTP_INVALID_MEDIA_TYPE",
    "error": "Unsupported Media Type",
    "message": "FST_ERR_CTP_INVALID_MEDIA_TYPE: Unsupported Media Type: application/x-www-form-urlencoded"
}
```

## Expected behavior/code
As for version `6.10.14` everything is parsed and working

## Environment
<pre><code>
Nest version: 6.10.14 -> 6.11.1

For Tooling issues:
- Node version: 10.16.3
- Platform: Amazon Linux/4.10.2

**Comment by AcidSlide** [comment]
@txantslusam shouldn't be reported as "Regression"??

**Comment by txantslusam** [comment]
@AcidSlide Yea, I think it should be regression. I edited my original issue to regresion template.

**Comment by kamilmysliwiec** [comment]
This should be fixed in 6.11.2. Let me know if you face any issues

**Comment by txantslusam** [comment]
@kamilmysliwiec Can confirm it works after update. Thanks!
